src/storage-folder.coffee

```diff
@@ -0,0 +1,27 @@
+path = require "path"
+fs = require "fs-plus"
+
+module.exports =
+class StorageFolder
+  constructor: (containingPath) ->
+    @path = path.join(containingPath, "storage")
+
+  store: (name, object) ->
+    fs.writeFileSync(@pathForKey(name), JSON.stringify(object), 'utf8')
+
+  load: (name) ->
+    try
+      stateString = fs.readFileSync(@pathForKey(name), 'utf8')
```

Comment:
Maybe, but I think we should make that it's own issue, just because it'd be a little bit of a behavior change from what we have now, and I think it'd take a little testing to make sure we didn't ever unintentionally wipe out the user's state.